docs/developer-guide.md

```diff
@@ -56,9 +56,11 @@ for starting out, just get either one working so you can play with the app.
 * When running on a physical device, if the device has Zulip installed
   from the Play Store, you may need to uninstall that version first.
 * Commands once you've set up:
+  * `react-native start` - start the packager which builds the JavaScript file
```

Comment:
I think this is potentially helpful, but you need to describe to the user why would they prefer that over `npm run ios-min` or the other alternatives.

Reply:
Huh -- in fact I've never used this command. Just `react-native run-android` works for me, and I do get the packager running.

So I have no idea why this would be helpful. Would be happy to learn, though. :)